Amplify/Categories/API/ClientBehavior/AmplifyAPICategory+ReachabilityBehavior.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Combine
+extension AmplifyAPICategory: APICategoryReachabilityBehavior {
+    @available(iOS 13.0, *)
+    public func reachabilityPublisher(for apiName: String? = nil) throws -> AnyPublisher<ReachabilityUpdate, Never>? {
```

Comment:
Now that we have a no-arg version of this method, we shouldn't assign a default value, or else it's hard to reason about which code path gets executed when invoking `reachabilityPublisher()`.